### PR TITLE
Fix AJAX navigation for compare link in `latest-tag-button` and add it to `view-markdown-source`

### PR DESCRIPTION
**Pull request by dertieran**
As pointed out by @kidonng in https://github.com/sindresorhus/refined-github/pull/4631#discussion_r682548229
The `#repo-content-pjax-container` did not work correctly for the compare link.

As a reparation I added the AJAX navigation to the `view-markdown-source` feature. 😄
I hope this will not conflict with #4639

## Test URLs

- Compare Link: https://github.com/sindresorhus/got
- `view-markdown-source`: https://github.com/sindresorhus/refined-github/blob/main/readme.md

## Screenshot

https://user-images.githubusercontent.com/9264728/128202068-c22975f2-f423-43dd-b0dd-0aafa2bee955.mp4

https://user-images.githubusercontent.com/9264728/128202284-cdce99f0-8345-4c95-b4bf-3829f7ba5519.mp4

### CODE REVIEW

File: source/features/latest-tag-button.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ async function init(): Promise<false | void> {
 				<a
 					className="btn btn-sm btn-outline tooltipped tooltipped-ne"
 					href={buildRepoURL(`compare/${latestTag}...${defaultBranch}`)}
-					data-pjax="#repo-content-pjax-container"
+					data-pjax="#js-repo-pjax-container"
```
Comment (kidonng) [inline source/features/latest-tag-button.tsx:139]: No, this won't work either

https://user-images.githubusercontent.com/44045911/128203855-b5f5b3dd-437c-4fc8-a905-ac4976f91517.mov

Comment (dertieran) [reply source/features/latest-tag-button.tsx:139]: Okay maybe I shouldn't do quick fixes for this 😅
But interesting that the next navigation is the issue in that case.

**Comment by dertieran** [comment]
Okay 👍🏻 
I guess we should still fix the bug asap even if it is just removing the attribute from that link.
So the new version doesn't get published with a known bug, thoughts @fregante?

**Comment by fregante** [comment]
No rush to publish, the bugs are not severe. I'd like to fix them before the next schedule release on September 1st though 😃

**Comment by kidonng** [comment]
Interestingly, the AJAX requests for compare page now returns document fragment correctly. I wonder if something changes in GitHub or it's just my previous test is flawed.

**Comment by fregante** [comment]
Has this been tested? Do you want to merge it as is?

**Comment by kidonng** [comment]
@fregante The fix is not needed anymore

**Comment by kidonng** [comment]
Oh, the `view-markdown-source` link improvement is still useful... But as discussed above I'm probably gonna merge it in my PR.

**Pull request closed without merging**